fixtures/eurolite/edx-4.json

```diff
@@ -1,11 +1,12 @@
 {
   "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
-  "name": "EDX-4",
+  "name": "EDX-4 4-channel DMX dimmer pack",
```

Comment:
The name should stay "EDX-4", the rest could go into `comment`.

Reply:
Done